angular/core/components/thread_page/thread_page_controller.coffee

```diff
@@ -3,7 +3,7 @@ angular.module('loomioApp').controller 'ThreadPageController', ($scope, $routePa
 
   @requestedProposalKey = $routeParams.proposal or $location.search().proposal
   @requestedCommentId   = parseInt($routeParams.comment or $location.search().comment)
-  $location.url($location.path())
+  $location.url($location.path()) unless $location.search().position
```

Comment:
Actually, it occurred to me that there are probably other times when we don't want to clear the url e.g. when a person is following a link in a membership request email (i.e. someone has requested access to one of their groups).